packages/broker/src/config/ConfigWizard.ts

```diff
@@ -1,254 +1,539 @@
-import path from 'path'
-import { writeFileSync, existsSync, mkdirSync, chmodSync } from 'fs'
+import { checkbox, confirm, input, password, select } from '@inquirer/prompts'
+import { config as cfg, config } from '@streamr/config'
+import { toEthereumAddress } from '@streamr/utils'
 import chalk from 'chalk'
-import { v4 as uuid } from 'uuid'
-
-import * as WebsocketConfigSchema from '../plugins/websocket/config.schema.json'
+import { BigNumber, Wallet, providers, utils } from 'ethers'
+import { isAddress } from 'ethers/lib/utils'
+import { chmodSync, existsSync, mkdirSync, writeFileSync } from 'fs'
+import { produce } from 'immer'
+import path from 'path'
+import { z } from 'zod'
+import { generateMnemonicFromAddress } from '../helpers/generateMnemonicFromAddress'
 import * as MqttConfigSchema from '../plugins/mqtt/config.schema.json'
+import * as WebsocketConfigSchema from '../plugins/websocket/config.schema.json'
 import * as BrokerConfigSchema from './config.schema.json'
-import { getDefaultFile } from './config'
-import { CURRENT_CONFIGURATION_VERSION, formSchemaUrl } from '../config/migration'
-import { generateMnemonicFromAddress } from '../helpers/generateMnemonicFromAddress'
-import { toEthereumAddress } from '@streamr/utils'
+import { ConfigFile, getDefaultFile } from './config'
 
-export interface PrivateKeyAnswers extends Answers {
-    generateOrImportPrivateKey: 'Import' | 'Generate'
-    importPrivateKey?: string
-}
+const MinBalance = utils.parseEther('0.1')
+
+export const start = async (): Promise<void> => {
+    log(`
+        >
+        > ***Welcome to the Streamr Network!***
+        > This Config Wizard will help you setup your node.
+        >
+        > The steps are documented here:
+        > *https://docs.streamr.network/guides/how-to-run-streamr-node#config-wizard*
+        >
+    `)
+
+    try {
+        const privateKey = await getPrivateKey()
+
+        const nodeAddress = new Wallet(privateKey).address
+
+        const network = await getNetwork()
+
+        const operator = await getOperatorAddress()
+
+        const operatorPlugins = operator ? {
+            operator: {
+                operatorContractAddress: operator,
+            },
+        } : {}
+
+        const { http, ...pubsubPlugins } = await getPubsubPlugins()
+
+        if (http) {
+            Object.assign(pubsubPlugins, {
+                http: {},
+            })
+        }
+
+        const storagePath = await getStoragePath()
+
+        const httpServer = http?.port ? { port: http.port } : void 0
+
+        const config: ConfigFile = {
+            client: {
+                auth: {
+                    privateKey,
+                },
+            },
+            plugins: {
+                ...operatorPlugins,
+                ...pubsubPlugins,
+            },
+            httpServer,
+        }
+
+        persistConfig(
+            storagePath,
+            network === 'polygon' ? config : getMumbaiConfig(config)
```

Comment:
Could introduce a string literal enum for networks, and also convert this ternary something like `if (network === 'mumbai') { config = getMumbaiConfig(config) }` so that possible third network will be handled correctly (if we add that at some point)